packages/nux/README.md

```diff
@@ -22,7 +22,7 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 See [the component's README][dot-tip-readme] for more information.
 
-[dot-tip-readme]: https://github.com/WordPress/gutenberg/tree/master/packages/nux/src/components/dot-tip/README.md
+[dot-tip-readme]: /packages/nux/src/components/dot-tip/README.md
```

Comment:
It doesn't look like we currently handle components of packages, ie. the `dot-tip` component doesn't exist in the WordPress.org handbook. I suspect the link to the readme is intentional here?

Reply:
Good catch, it shouldn't probably. I will revert this line :)